README.md

```diff
@@ -18,19 +18,18 @@ And then execute:
 
 Before starting this, you'll need to:
 
-- [Read the documentation](https://docs.publishing.service.gov.uk/manual/ab-testing.html) for an overview on how a/b testing works on GOV.UK. 
+- [Read the documentation](https://docs.publishing.service.gov.uk/manual/ab-testing.html) for an overview on how a/b testing works on GOV.UK.
 - The cookie and header name in [govuk-cdn-config](https://github.com/alphagov/govuk-cdn-config/blob/master/ab_tests/ab_tests.yaml) must match the test name parameter that you pass to the Gem. The cookie name is case-sensitive.
 
 ## Usage
 
-### Outline 
+### Outline
 
 To enable testing in the app, your Rails app needs:
 
 1. [Some piece of logic to be A/B tested](#1-example-ab-test-logic)
 2. [A response HTTP header that tells Fastly you're doing an A/B test](#2-http-response-header-to-fastly)
-3. [A HTML meta tag that will be used to measure the results, and which specifies
-   the dimension to use in Google Analytics](#3-add-html-metatag-tags-to-your-layouts)
+3. [A HTML meta tag that will be used to measure the results in Google Analytics](#3-add-html-metatag-tags-to-your-layouts)
```

Comment:
Do we still have value having the meta tag without the dimension? I've assumed (potentially incorrectly) that the primary purpose of the meta tag was for getting data into GA and not sure if that'd be problematic or superfluous without dimension.

Reply:
Please, see my comment below.